db/sql/blocks/aggregateBlocksReward.sql

```diff
@@ -11,7 +11,7 @@ WITH delegate AS
      AND m."publicKey" = decode($1, 'hex')
    LIMIT 1),
      rewards AS
-  (SELECT count(1) AS count,
+  (SELECT count(*) AS count,
```

Comment:
Please remove `AS count`, as it is pointless to add an alias the same as the result.